app/controllers/editor/categories_controller.rb

```diff
@@ -43,6 +43,24 @@ def destroy
     redirect_to editor_categories_url, notice: "カテゴリが削除されました"
   end
 
+  # move to left
+  def move_higher
```

Comment:
It is close but not correct.
* 👍  Create some controller `Editor::Categories::XXXController#update`
* 👎  The controller name is not noun

The controller name should be a noun.
So the name will be `PositionController`, `OrderController` or something.